### DISCUSSION TITLE
Prácticas de preparación

### DISCUSSION THREAD

**Discussion by eskaider** [Launch-X-Latam/MisionFrontEnd#128] [Q&A]
Muy buen día tengan compañeros el día viernes estuve buscando el enlace a las prácticas por teams pero por bastantes comentarios no lo encuentro alguien ¿puede compartirme el enlace para poder comenzar con las prácticas?

**Comment by D3r3-k** [comment]
La practica esta en la Mision FrontEnd de aqui en GitHub, entra al primer modulo y en la lista de temas, de ultimo esta la practica

**Comment by dmejia96** [comment]
Buen día! Anexo enlace.
https://github.com/Launch-X-Latam/MisionFrontEnd/blob/main/01%20-%20INTRO/practicas/README.md
Saludos!
